payments/card/direct.md

```diff
@@ -28,53 +28,71 @@ sidebar:
                       body="This section of the Developer Portal is under
                       review and may be incomplete or contain minor errors." %}
 
-{% include jumbotron.html body="The Direct Payment scenario **is used by
-customers that are compliant with PCI-DSS regulations**, and is a way to
-implement card payments without using Swedbank Pay hosted payment page." %}
+{% include jumbotron.html body="The Direct Card Payment scenario is used by
+customers that are **compliant with PCI-DSS regulations**, and is a way to
+implement Card Payments without using a Swedbank Pay hosted payment page." %}
+
+{% include alert.html type="neutral" icon="info" header="Note on 3-D Secure
+Authentication" body="Although there is no need to redirect to a Swedbank Pay
+hosted payment page in the Direct Card Payment scenario, consumers will need to
+be **redirected to a 3-D Secure page hosted by the issuing bank if the bank
+requires it**. Whether 3-D Secure authentication is required for every payment
+is up to each issuing bank and out of Swedbank Pay's control." %}
 
 {% include alert.html type="danger" icon="error" header="PCI-DSS Complicance"
 body="The direct integration option requires you to collect the card data on
 your website, which means it must be [PCI-DSS Compliant](
 https://www.pcisecuritystandards.org/)." %}
 
-## Introduction
-
-* The payer places an order and you make a `POST` request towards Swedbank Pay
-  with gathered `Purchase` information.
-* The action taken next is the `direct-authorization` operation that is returned
-  in the first request. You `POST` the payer's card data to the URL in
-  the[`direct-authorization` operation][authorization].
-* If 3-D Secure authentication is required, you will then receive a URL where
-  you will have to redirect the payer.
-* When the payment is completed, the payer needs to be redirected back to your
-  merchant/webshop site.
-* Finally you make a `GET` request towards Swedbank Pay with the `paymentID`
-  received in the first step, which will return the purchase result.
+## Summary
```

Comment:
maybe we can call this Payment flow ? It would be nice if it was similiar to all the other sections?

Reply:
Instead of "Summary", you mean?